### DISCUSSION TITLE
Matching Cloudlog and LOTW Locations

### DISCUSSION THREAD

**Discussion by petebrunelli** [magicbug/Cloudlog#1741] [Q&A] [answered]
As I add operating locations to Cloudlog it looks like there needs to be a corresponding Station Location in LOTW if I want the QSOs to match. If I operate from a different 6-digit grid with FN41, and I use separate Cloudlog locations to stay organized an keep the ability to calculate real distances, how does LOTW deal with that? Can anyone comment with some clarity on how these need to match? Name? Grid? 

Thanks, Pete N1QDQ

**Comment by phl0** [comment, marked as answer]
You would just have to setup another station location in Cloudlog and that's all (in case you upload your QSOs to LotW from Cloudlog directly). 
If you export to adi(f) and upload using tqsl you have to define station profiles for each grid.

**Comment by petebrunelli** [comment]
Are you saying that Cloudlog creates the new location in LOTW, or that it doesn't check locations? Let's say I have multiple locations within my home grid of FN41. I create a separate location for each in Cloudlog to track where I operated from and when. Does LOTW consider them all "FN41" and not individual operating locations? I have a few thousand non-satellite contacts in LOTW and haven't been to fastidious about the subset that weren't from my home QTH. But on the sats I am trying to keep the log as accurate as possible. Thanks. Pete
